osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SecretManagementService.java

```diff
@@ -35,147 +44,318 @@
 
 @Service
 @Slf4j
-public class SecretManagementService implements SecurityKeyService {
+/**
+ * Service for storing, activating and retrieving device keys.
+ * Also performs RSA encryption/decryption operations for SOAP messaging purposes.
+ */
+public class SecretManagementService { //implements SecurityKeyService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)